scripts/build.py

```diff
@@ -1988,6 +1988,12 @@ def _form_env(self):
 
         env['PATH'] = paths + os.path.pathsep + env['PATH']
 
+        # This trickery is needed for the bash to properly inherit the PATH that we've set here
+        # See /etc/profile for further details
+        is_mingw = env.get('MSYSTEM', '') in ('MINGW64',)
+        if is_mingw:
+            env['ORIGINAL_PATH'] = env['PATH']
+
```

Comment:
WIthout this, `git bash` wouldn't find `waf` when you wanted to run it stand alone on the command line. (E.g. when rebuilding a single library)